ui/src/data_explorer/components/VisHeader.js

```diff
@@ -3,21 +3,19 @@ import classNames from 'classnames'
 
 const VisHeader = ({views, view, onToggleView, name}) => (
   <div className="graph-heading">
-    <div className="graph-actions">
-      {views.length
-        ? <ul className="toggle toggle-sm">
-            {views.map(v => (
-              <li
-                key={v}
-                onClick={() => onToggleView(v)}
-                className={classNames('toggle-btn ', {active: view === v})}
-              >
-                {v}
-              </li>
-            ))}
-          </ul>
-        : null}
-    </div>
+    {views.length
+      ? <ul className="toggle toggle-sm">
+          {views.map(v => (
+            <li
+              key={v}
+              onClick={() => onToggleView(v)}
+              className={classNames('toggle-btn ', {active: view === v})}
```

Comment:
prefer `classnames` here (and throughout file & PR), personally, to help disambiguate the two (the latter is not a React utility)